config/src/main/java/com/typesafe/config/impl/Tokenizer.java

```diff
@@ -52,6 +52,14 @@ else if (Character.isISOControl(codepoint))
         return new TokenIterator(origin, input, flavor != ConfigSyntax.JSON);
     }
 
+    static String render(Iterator<Token> tokens) {
+        String renderedText = "";
```

Comment:
@havocp Should we modify `tokenize`, then, to return an iterable rather than an iterator? Or do we expect the user to use the iterator to construct some sort of collection of tokens and pass that into `render`?